air/src/execution_step/execution_context/instruction_error/instruction_error_definition.rs

```diff
@@ -74,53 +74,57 @@ pub(crate) fn error_from_raw_fields(error_code: i64, error_message: &str, instru
 /// Checks that a scalar is a value of an object types that contains at least two fields:
 ///  - error_code
 ///  - message
-pub(crate) fn check_error_object(scalar: &JValue) -> Result<(), LastErrorObjectError> {
+pub(crate) fn check_error_object(scalar: &JValue) -> Result<(), ErrorObjectError> {
     let fields = match scalar {
         JValue::Object(fields) => fields,
-        _ => return Err(LastErrorObjectError::ScalarMustBeObject(scalar.clone())),
+        _ => return Err(ErrorObjectError::ScalarMustBeObject(scalar.clone())),
     };
 
     let check_field = |field_name| {
         fields
             .get(field_name)
-            .ok_or_else(|| LastErrorObjectError::ScalarMustContainField {
+            .ok_or_else(|| ErrorObjectError::ScalarMustContainField {
                 scalar: scalar.clone(),
                 field_name,
             })
     };
 
     let error_code = check_field(ERROR_CODE_FIELD_NAME)?;
-    ensure_jvalue_is_integer(scalar, error_code, ERROR_CODE_FIELD_NAME)?;
+    ensure_jvalue_is_integer_and_not_0(scalar, error_code, ERROR_CODE_FIELD_NAME)?;
 
     let message = check_field(MESSAGE_FIELD_NAME)?;
     ensure_jvalue_is_string(scalar, message, MESSAGE_FIELD_NAME)?;
 
     Ok(())
 }
 
-fn ensure_jvalue_is_integer(
+fn ensure_number_is_not_0(number: i64) -> Result<(), ErrorObjectError> {
```

Comment:
Let's call it `ensure_error_code_is_error`, because 0 is not represent intention, it says only that the function checks it against 0, but doesn't say what this 0 mean.